std/utf.d

```diff
@@ -2519,9 +2519,15 @@ ubyte codeLength(C)(dchar c) @safe pure nothrow @nogc
     in a string whose character type is $(D C). This is particularly useful
     when slicing one string with the length of another and the two string
     types use different character types.
+
+    Params:
+        C = the character type to get the encoding length for
+        input = the input range to compute
+    Returns:
```

Comment:
How about:

- the input range to use for the encoding length computation
- the input range to calculate the encoding length from